discovery-provider/src/tasks/user_library.py

```diff
@@ -94,6 +96,11 @@ def user_library_state_update(
 
     return num_total_changes
 
+######## HELPERS ########
+
+def dispatch_favorite(session, bus, save, block_number):
```

Comment:
for all of these it seems super unlikely that the redis op in bus.dispatch fails but if it does for whatever reason we could throw (and likely wouldn't be universal). what do you think the optimal behavior would be here?

Reply:
yeah it's a good question. I think for now I'd just add a try / catch around this, from a system level though we'd rely on the quorum if a single DP gets in a weird state. Notably, even if we miss a single follow for ex, if that user ever follows another user again we'd recompute and get both.

Reply:
interesting, that is pretty cool